config.example.js

```diff
@@ -1,5 +0,0 @@
-module.exports = {
```

Comment:
This file should not be deleted, as it will give users an idea on how to construct their config file.